### DISCUSSION TITLE
Unraid | IP of container changes, Plex Trakt sync stops working

### DISCUSSION THREAD

**Discussion by hanses89** [Taxel/PlexTraktSync#849] [Support] [answered]
I’m running Plex and Plex Trakt sync on unraid and they are running in containers.. I use a custom docker network for all my containers to run on. Sometimes when the server is restarted the container for Plex might change IP, and Plex Trakt sync stops working. I start the container but it stops right away. I tried to change the IP in the .env file to match the new IP of Plex, but it still won’t work… the “fix” is to delete the container and start fresh… this is really a pain in the ass.  Any tips?

**Comment by glensc** [comment]
converted to a discussion, since you ignored the issue templates we have in this project.

**Reply by hanses89** [reply]
Sorry about that! I’m new at GitHub. Won’t happen again:)

**Reply by glensc** [reply]
your discussion title is not visible either, no details about unraid in it (edit it)

**Comment by glensc** [comment, marked as answer]
You can try this patch:
- https://github.com/Taxel/PlexTraktSync/pull/850/files

**Comment by glensc** [comment]
You may have some luck putting the containers to the same network so you can refer to other containers with their names rather IP addresses.

With docker-compose (no clues about unraid):

- https://docs.docker.com/compose/networking/

perhaps the external network example.

**Reply by hanses89** [reply]
I have done that. All my containers is on a custom docker network so they can communicate by name. But changing the local url in the .env file does not fix the issue. Neither does changing the url to the correct new ip of Plex.. so I don't understand what I can do here. There is no way to set this prior to the first sync it seems?

**Reply by glensc** [reply]
did you read the other reply?
- https://github.com/Taxel/PlexTraktSync/discussions/849#discussioncomment-2447783

but back to using container names: perhaps you need to be concrete here, show actual file contents (obfuscate if needed), show ping, etc commands with their output.
